FirebaseABTesting/CHANGELOG.md

```diff
@@ -1,8 +1,8 @@
 # v3.2.0
 - [added] Added completion handler for FIRExperimentController's updateExperimentsWithServiceOrigin method.
 - [deprecated] Deprecated `FIRExperimentController.updateExperiments(serviceOrigin:events:policy:lastStartTime:payloads:)`.
-- [added] Added method to expire experiments that are no longer running.
-- [added] Added method to directly activate an experiment.
+- [added] Added `FIRExperimentController.validateRunningExperiments(serviceOrigin:runningExperimentPayloads:)` to expire experiments that are no longer running.
```

Comment:
nit: rephrase this and below to `, allowing callers to expire experiments that are no longer running.` or something similar.